core/src/main/java/org/polypheny/db/catalog/entity/CatalogPartitionPlacement.java

```diff
@@ -35,6 +41,15 @@
     public final String physicalTableName;
 
 
+    // Related to multi-tier replication. If Placement is considered a primary node it needs to receive every update eagerly.
+    // If false, nodes are considered refreshable and can be lazily replicated
+    // This attribute is derived from an effective data placement (table entity on a store)
+    // This means that the store is not entirely considered a primary and can therefore be different on another table
+    // TODO @HENNLO remove default role
+    public final DataPlacementRole role = DataPlacementRole.UPTODATE;
+    public Timestamp updateTimestamp;
```

Comment:
All catalog objects should be immutable.

See: https://mapdb.org/book/quick-start/

> Data stored in MapDB (keys and values) should be immutable. MapDB serializes objects on background.